cookbooks/swift/recipes/configs.rb

```diff
@@ -166,7 +166,12 @@
     source "/etc/swift/#{filename}.erb"
     owner node["username"]
     group node["username"]
-    variables({}.merge(node))
+    variables({
+        :auth_uri => node['auth_uri'],
+        :storage_policies => node['storage_policies'],
+        :username => node['username'],
+        :base_uri => node['base_uri'],
+      })
```

Comment:
maybe explicit is better than implicit - but this was trying to make ALL node variables available to templates (like **node)... was that wrong?